faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -35,6 +42,30 @@ object Decoder {
       }
   }
 
+  implicit object CharDecoder extends Decoder[Char] {
```

Comment:
Never mind. I thought a `Char` would be encoded as a `String` when I wrote that. The encoders are below and they are consistent with the decoders.